### PR TITLE
Adds Slime Feeding

### PR DESCRIPTION
**Pull request by Cameron653**
- Allows slimes to feed people
- Think succubus drain but in reverse
- If the slime runs out of nutrition & drain finalization is on, they disappear into whatever gut the person they're feeding has selected.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/station/station_special_abilities_vr.dm [hunk 1]
```diff
@@ -606,15 +606,80 @@
...
+	for(var/stage = 1, stage<=100, stage++) //100 stages.
+		switch(stage)
+			if(1)
+				to_chat(C, "<span class='notice'>You begin to feed [T]...</span>")
+				to_chat(T, "<span class='notice'>An odd sensation flows through your body as [C] begins to feed you!</span>")
+				T.nutrition = (T.nutrition + (C.nutrition*0.05)) //Drain a small bit at first. 5% of the prey's nutrition.
+				C.nutrition = C.nutrition*0.95
+			if(2)
+				to_chat(C, "<span class='notice'>You feel weaker with every passing moment of feeding [T].</span>")
+				src.visible_message("<span class='notice'>[C] seems to be doing something to [T], [T]'s body looking stronger with every passing moment!</span>")
```
Comment (Cyantime) [inline code/modules/mob/living/carbon/human/species/station/station_special_abilities_vr.dm]: `[T], [T]` could probably be rephrased in a way that isn't repeating the target's name twice like that

**Comment by Screemonster** [comment]
... don't think I'm going to let that branch name slip by without a comment.

**Comment by Cameron653** [comment]
Changed [T], [T]'s to [T], resulting in [T]'s

**Comment by SpadesNeil** [comment]
>Changed [T], [T]'s to [T], resulting in [T]'s

https://www.youtube.com/watch?v=KufiqCh3Qd4

**Comment by Cameron653** [comment]
I edited the way it looks.
Instead of: "Person seems to be doing something to Zee, Zee's body looking...
It does: "Person seems to be doing something to Zee, resulting in Zee's body looking...

**Pull request merged**